src/main/java/com/diffmin/util/SpoonUtil.java

```diff
@@ -73,19 +75,27 @@ public static CtModel buildModel(File file) throws FileNotFoundException {
         return launcher.buildModel();
     }
 
-    /**
-     * Pretty prints the model.
-     *
-     * @param model model to be pretty printed
-     * @return patched program
-     */
-    public static String displayModifiedModel(CtModel model) {
-        CtType<?> firstType = model.getAllTypes().stream().findFirst().get();
-        CtCompilationUnit cu = firstType.getFactory().CompilationUnit().getOrCreate(firstType);
+    public static String prettyPrintModel(CtModel model) {
+        List<CtCompilationUnit> compilationUnits =
+                List.copyOf(
+                        model.getUnnamedModule().getFactory().CompilationUnit().getMap().values());
+        List<CtCompilationUnit> modelCuCandidates =
+                compilationUnits.stream()
+                        .filter(cu -> !cu.getFile().getName().equals("package-info.java"))
+                        .collect(Collectors.toList());
```

Comment:
This filtering should not be necessary, if there's a `package-info.java` compilation unit, something has gone wrong. I fetched this branch and removed this check, all tests still pass.

Reply:
>  if there's a package-info.java compilation unit, something has gone wrong.

Yeah, I checked now. It was not necessary. However, something was going wrong on `type-insertion` branch. But of course, I will get to that later.